### ISSUE TITLE
support for multilabel datasets

### ISSUE THREAD

**Issue by davsol** [dmlc/gluon-cv#307]
Hi,
Thanks for this great repo!
It's so intuitive and flexible, with so much different algorithms to play with :)

I am wondering what's your the best solution to support multilabel datasets for detection.

Part of YoloV3's loss is the decision to decouple class predictions, by using binary cross-entropy instead of softmax. This enables training multilabel datasets (ex - where the classes can be - "man", "human")

Correct me if I'm wrong, but current datasets cannot actually output a multilabel tagged object, since labels are saved in a NxL size (N- batch size, L is constant - 6 for Pascal)

Do you have plans to support this?

**Comment by zhreshold** [comment]
in case you have total length M of multilables, you can give the labels a new shape (B, N, 4 + M), where B is the batch size, N is the maximum number of objects in each image. 
This may get quite big, but logic is still the same in gluoncv.
